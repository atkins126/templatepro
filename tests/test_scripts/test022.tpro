DataSets
{{if customers}}
{{for cust in customers}}
Code: {{:cust.Code}} - Name: {{:cust.Name}}
SINGLE {{:customer.Code}} {{:customer.Name}}
{{endfor}}
{{endif}}
