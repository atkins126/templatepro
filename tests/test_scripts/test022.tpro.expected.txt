DataSets
Code: 1 - Name: Ford
SINGLE 1 Ford
Code: 2 - Name: Ferrari
SINGLE 1 Ford
Code: 3 - Name: Lotus
SINGLE 1 Ford
Code: 4 - Name: FCA
SINGLE 1 Ford
Code: 5 - Name: Hyundai
SINGLE 1 Ford
Code: 6 - Name: De Tomaso
SINGLE 1 Ford
Code: 7 - Name: Dodge
SINGLE 1 Ford
Code: 8 - Name: Tesla
SINGLE 1 Ford
Code: 9 - Name: Kia
SINGLE 1 Ford
Code: 10 - Name: Tata
SINGLE 1 Ford
Code: 11 - Name: Volkswagen
SINGLE 1 Ford
Code: 12 - Name: Audi
SINGLE 1 Ford
Code: 13 - Name: Skoda
SINGLE 1 Ford
